webdriver/tests/bidi/storage/set_cookie/partition.py

```diff
@@ -5,30 +5,23 @@
 pytestmark = pytest.mark.asyncio
 
 
-async def test_partition_context(bidi_session, set_cookie, top_context, test_page, origin, domain_value):
+async def test_partition_context(bidi_session, set_cookie, top_context, test_page, domain_value):
     await bidi_session.browsing_context.navigate(context=top_context["context"], url=test_page, wait="complete")
 
-    source_origin = origin()
-    partition = BrowsingContextPartitionDescriptor(top_context["context"])
-
     set_cookie_result = await set_cookie(
         cookie=create_cookie(domain=domain_value()),
-        partition=partition)
+        partition=(BrowsingContextPartitionDescriptor(top_context["context"])))
```

Comment:
we could also test that the explicitly specified sourceOrigin should be respected, that is, without the BrowsingContextPartitionDescriptor.

Reply:
Do you mean the [`test_partition_storage_key_source_origin`](https://github.com/web-platform-tests/wpt/blob/fcc993df93c864e0528939f9fb394fc3bf272186/webdriver/tests/bidi/storage/set_cookie/partition.py#L46)?

Reply:
yes, that's right. Sounds like it's covered already.